src/Cake.Core/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Cake.Core
+{
+    internal static class Constants
+    {
+        public static class Paths
```

Comment:
@patriksvensson when you say Fixed here, what exactly do you mean?  Did you mean that you have added the existing Configuration value of the static Constants class?  If so, I don't see that having been done.